src/sentry/south_migrations/0348_fix_project_key_rate_limit_window_unit.py

```diff
@@ -0,0 +1,917 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        db.commit_transaction()
+        try:
+            self._forwards(orm)
+        except Exception:
+            db.start_transaction()
+            raise
+        db.start_transaction()
+
+    def _forwards(self, orm):
+        from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+        ProjectKey = orm['sentry.ProjectKey']
+
+        queryset = ProjectKey.objects.filter(rate_limit_window__isnull=False)
+        for key in RangeQuerySetWrapperWithProgressBar(queryset):
+            ProjectKey.objects.filter(pk=key.pk).update(
+                rate_limit_window=key.rate_limit_window * 60)
```

Comment:
Just as a little bit of protection from race here:

```python
ProjectKey.objects.filter(pk=key.pk, rate_limit_window=key.rate_limit_window).update(rate_limit_window=key.rate_limit_window * 60)
```

Just in case for the small chance it were updated in between, we don't clobber the value.